### DISCUSSION TITLE
Can't install livewire 3

### DISCUSSION THREAD

**Discussion by yudha-dev** [livewire/livewire#5878] [4. General]
zsh: no matches found: livewire/livewire:^3.0@beta

**Comment by PhiloNL** [comment]
Are you running the entire command?

```shell
composer require livewire/livewire:^3.0@beta
```

**Reply by yudha-dev** [reply]
yes this via composer command not working but i install via package.json its working

**Comment by rafwoski** [comment]
i cant install it either.  

Win 11 + composer 2.5.8. 

Fresh Laravel + Breeze install.

![Screenshot 2023-07-22 160001](https://github.com/livewire/livewire/assets/106398573/887d6f2e-a0f4-4747-8051-77e701e815ac)

**Reply by nuernbergerA** [reply]
this should be fixed with beta.3 for now you can try dev-master

**Comment by rafwoski** [comment]
working with dev-master. Thanks for your help.

**Comment by jeremiahsherrill** [comment]
composer require livewire/livewire:dev-master
